proj4/src/main/scala/geotrellis/proj4/io/wkt/WKT.scala

```diff
@@ -22,7 +22,7 @@ import scala.io.Source
 
 object WKT {
   private val wktResourcePath = "/proj4/wkt/epsg.properties"
-  lazy val parsed: Map[Int, WktCS] = records.mapValues(WKTParser.apply)
+  lazy val parsed: Map[Int, WktCS] = records.mapValues(WKTParser.apply).toMap
```

Comment:
`mapValues` is deprecated in 2.13 

> Use .view.mapValues(f). A future version will include a strict version of this method (for now, .view.mapValues(f).toMap).

Reply:
Yes -- this is just to maintain backwards compatibility with 2.11 and 2.12 with the fewest changes possible.  There are hundreds if not thousands of deprecation warnings when compiling with 2.13 😬